www/json/connectionConfig.json

```diff
@@ -0,0 +1,15 @@
+{
+    "connectUrl": "http://52.91.65.149:8080",
+    "android": {
+        "auth": {
+            "method": "token_list",
+            "clientID": "8QU4GCAZ"
```

Comment:
dunno why the `clientID` is set, where you got the value from and what it is supposed to mean